### DISCUSSION TITLE
Add education platforms?

### DISCUSSION THREAD

**Discussion by FrancescoXX** [FrancescoXX/free-Web3-resources#42] [Ideas]
Should we add the education platforms section?

**Comment by suitupstan** [comment]
Yesss! I think that is a great idea! A lot of education platforms are in the communities section, polluting the area and making it difficult to navigate!

**Comment by FrancescoXX** [comment]
we need examples

**Comment by Shukla007** [comment]
I have added the education platform section 👍

**Comment by FrancescoXX** [comment]
thank you
